python/src/scippneutron/file_loading/load_nexus.py

```diff
@@ -26,6 +26,7 @@
 nx_sample = "NXsample"
 nx_source = "NXsource"
 nx_detector = "NXdetector"
+nx_chopper = "NXdisk_chopper"
```

Comment:
Consider calling this `nx_disk_chopper` or similar to disambiguate from fermi choppers etc which we may have to support later? This would also keep it consistent with nexus class names

Reply:
The name is set by the Nexus standard i.e. https://manual.nexusformat.org/classes/base_classes/NXdisk_chopper.html?highlight=nxdisk_chopper but maybe i misunderstand your comment. You are correct though, the standard does support https://manual.nexusformat.org/classes/base_classes/NXfermi_chopper.html#nxfermi-chopper

Reply:
Sorry, perhaps my comment wasn't entirely clear - I was suggesting renaming the python variable to match the nexus class name

```suggestion
nx_disk_chopper = "NXdisk_chopper"
```